CipherText/Rotator/Rotator.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jjose00
+ * Date: 11/7/17
+ * Time: 8:11 AM
+ */
+
+namespace Giftcards\Encryption\CipherText\Rotator;
+
+use Giftcards\Encryption\Encryptor;
+
+class Rotator
+{
+    /**
+     * @var Encryptor
+     */
+    private $encryptor;
+    /**
+     * @var StoreRegistry
+     */
+    private $storeRegistry;
+
+    public function __construct(Encryptor $encryptor, StoreRegistry $storeRegistry)
+    {
+        $this->encryptor = $encryptor;
+        $this->storeRegistry = $storeRegistry;
+    }
+
+    public function rotate($newProfile, $storeName, Bounds $bounds, ObserverInterface $observer)
```

Comment:
im thinking the argument order would make more sense as `$storeName, $newProfile, Bounds $bounds, ObserverInterface $observer`